modules/fswatch/fswatch_profile_linux.go

```diff
@@ -6,17 +6,17 @@
 
 package fswatch /* import "mig.ninja/mig/modules/fswatch" */
 
-var localFsWatchProfile = fsWatchProfile{
-	[]fsWatchProfileEntry{
-		{"/boot", []fsWatchObject{}},
-		{"/etc/cron.d", []fsWatchObject{}},
-		{"/var/spool/cron", []fsWatchObject{}},
-		{"/bin", []fsWatchObject{}},
-		{"/sbin", []fsWatchObject{}},
-		{"/usr/bin", []fsWatchObject{}},
-		{"/usr/sbin", []fsWatchObject{}},
-		{"/etc", []fsWatchObject{}},
-		{"/etc/init.d", []fsWatchObject{}},
-		{"/etc/systemd/system", []fsWatchObject{}},
+var localFsWatchProfile = profile{
+	entries: []profileEntry{
+		{path: "/boot", recursive: false},
+		{path: "/etc/cron.d", recursive: false},
+		{path: "/var/spool/cron", recursive: false},
+		{path: "/bin", recursive: false},
+		{path: "/sbin", recursive: false},
+		{path: "/usr/bin", recursive: false},
+		{path: "/usr/sbin", recursive: false},
+		{path: "/etc", recursive: true},
```

Comment:
I suspect this covers /etc/systemd and /etc/init.d and /etc/cron.d with recursive=true
also, /lib /usr/lib /lib64 /usr/lib64 are similar to /bin /usr/bin etc (i.e. might want to watch them too)